e2e/cypress/integration/interactive_menu/basic_options_spec.js

```diff
@@ -30,6 +30,8 @@ let longUsername;
 
 describe('Interactive Menu', () => {
     before(() => {
+        cy.requireWebhookServer();
+
```

Comment:
Getting 2 failures in this test
![Screen Shot 2020-03-10 at 6 51 00 PM](https://user-images.githubusercontent.com/487991/76374495-9d64cc00-6300-11ea-895e-e31b60450b44.png)
![Screen Shot 2020-03-10 at 6 51 24 PM](https://user-images.githubusercontent.com/487991/76374499-9fc72600-6300-11ea-9456-b869786dde1b.png)